Changing project on device doesn't remove old modules

### Current Behavior

When I change the project that my device is running and deploy a new snapshot for that project Node-RED still picks up and tries to install modules from the previous project.
I think this is due to `/opt/flowforge-device/project/node_modules` not being wiped.

### Expected Behavior

Deploying a new snapshot should result in a clean install of only the modules in the snapshpot

### Steps To Reproduce

Build project with a custom node.
Deploy to device,

Build new project without custom node.
Deploy to device.

Original custom node is still started.

### Environment

- FlowForge version: 0.8.0
- Agent 0.2.0
- Node.js version: 16.16
- npm version:
- Platform/OS:
- Browser:

@ben NPM 8.11.0

The package.json only has the project nodes, the sense-hat node and node-red which is what I would expect for this project/snapshot. But as you can see when it starts up its saying missing node modules for morse and serial port which were in the previous project.
```
pi@sensehatpi:/opt/flowforge-device/project $ flowforge-device-agent 
[2022-08-12T14:00:20.433Z] INFO: FlowForge Device Agent
[2022-08-12T14:00:20.440Z] INFO: Version: 0.2.0
[2022-08-12T14:00:20.440Z] INFO: Device: nEjxwoxM91
[2022-08-12T14:00:20.441Z] INFO: ForgeURL: https://forge.flowforge.dev
[2022-08-12T14:00:20.460Z] INFO: Project: 10f621bc-88c1-4461-94f0-ca0ae81dff4e
[2022-08-12T14:00:20.461Z] INFO: Snapshot: awbNlYPn40
[2022-08-12T14:00:20.462Z] INFO: Settings: d5f76bbffc9252f34a7c24aa6f6d919d9de932981800f3eea51245dd8149dba6
[2022-08-12T14:00:20.700Z] INFO: Starting heartbeat thread. Interval: 30s
[2022-08-12T14:00:21.356Z] INFO: Starting project
[NR] 12 Aug 15:00:23 - [info] 
[NR] Welcome to Node-RED
[NR] ===================
[NR] 12 Aug 15:00:23 - [info] Node-RED version: v3.0.1
[NR] 12 Aug 15:00:23 - [info] Node.js  version: v16.16.0
[NR] 12 Aug 15:00:23 - [info] Linux 5.15.32-v7+ arm LE
[NR] 12 Aug 15:00:25 - [info] Loading palette nodes
[NR] 12 Aug 15:00:27 - [warn] Missing node modules:
[NR] 12 Aug 15:00:27 - [warn]  - node-red-node-serialport (1.0.1): serial-port, serial out, serial in, serial request
[NR] 12 Aug 15:00:27 - [warn]  - node-red-contrib-morse (0.0.9): morse-out
[NR] 12 Aug 15:00:27 - [info] Settings file  : /opt/flowforge-device/project/settings.js
[NR] 12 Aug 15:00:27 - [info] Context store  : 'default' [module=memory]
[NR] 12 Aug 15:00:27 - [info] Installing module: node-red-node-serialport, version: 1.0.1
[NR] 12 Aug 15:00:27 - [info] User directory : /opt/flowforge-device/project
[NR] 12 Aug 15:00:27 - [warn] Projects disabled : set editorTheme.projects.enabled=true to enable
[NR] 12 Aug 15:00:27 - [info] Flows file     : /opt/flowforge-device/project/flows.json
[NR] 12 Aug 15:00:27 - [info] Admin UI disabled
[NR] 12 Aug 15:00:27 - [info] Server now running at http://127.0.0.1:1880
[NR] 12 Aug 15:00:27 - [warn] Encrypted credentials not found
[NR] 12 Aug 15:00:27 - [info] Starting flows
[NR] 12 Aug 15:00:27 - [info] Started flows
[NR] 12 Aug 15:00:27 - [info] Project Link nodes connected
[NR] 12 Aug 15:00:33 - [warn] Installation of module node-red-node-serialport failed:
[NR] 12 Aug 15:00:33 - [warn] ------------------------------------------
[NR] 12 Aug 15:00:33 - [warn] npm WARN config production Use `--omit=dev` instead.
[NR] npm ERR! code 127
[NR] npm ERR! path /opt/flowforge-device/project/node_modules/@serialport/bindings-cpp
[NR] npm ERR! command failed
[NR] npm ERR! command sh -c node-gyp-build
[NR] npm ERR! sh: 1: node-gyp-build: not found
[NR] npm ERR! A complete log of this run can be found in:
[NR] npm ERR!     /home/pi/.npm/_logs/2022-08-12T14_00_29_081Z-debug-0.log
[NR] 12 Aug 15:00:33 - [warn] ------------------------------------------
```
Those modules are still present in node-modules which is where i assume Node-RED is finding them?

--- a/lib/launcher.js
+++ b/lib/launcher.js
@@ -43,6 +43,8 @@ class Launcher {
         const packageJSON = JSON.parse(JSON.stringify(packageJSONTemplate))
         packageJSON.dependencies = JSON.parse(JSON.stringify(this.snapshot.modules))
         await fs.writeFile(this.files.packageJSON, JSON.stringify(packageJSON, ' ', 2))
+        await fs.unlink(path.join(this.projectDir, '.config.nodes.json'))
+        await fs.unlink(path.join(this.projectDir, '.config.nodes.json.backup'))
     }
 
     async installDependencies () {

--- a/lib/launcher.js
+++ b/lib/launcher.js
@@ -43,8 +43,8 @@ class Launcher {
         const packageJSON = JSON.parse(JSON.stringify(packageJSONTemplate))
         packageJSON.dependencies = JSON.parse(JSON.stringify(this.snapshot.modules))
         await fs.writeFile(this.files.packageJSON, JSON.stringify(packageJSON, ' ', 2))
-        await fs.unlink(path.join(this.projectDir, '.config.nodes.json'))
-        await fs.unlink(path.join(this.projectDir, '.config.nodes.json.backup'))
+        await fs.rm(path.join(this.projectDir, '.config.nodes.json'), { force: true })
+        await fs.rm(path.join(this.projectDir, '.config.nodes.json.backup'), { force: true})
     }
 
     async installDependencies () {

